src/diffusers/models/unets/unet_motion_model.py

```diff
@@ -1080,10 +1066,19 @@ def forward(
             if cross_attention_kwargs.get("scale", None) is not None:
                 logger.warning("Passing `scale` to `cross_attention_kwargs` is deprecated. `scale` will be ignored.")
 
-        hidden_states = self.resnets[0](hidden_states, temb)
+        hidden_states = self.resnets[0](input_tensor=hidden_states, temb=temb)
 
         blocks = zip(self.attentions, self.resnets[1:], self.motion_modules)
         for attn, resnet, motion_module in blocks:
+            hidden_states = attn(
+                hidden_states=hidden_states,
+                encoder_hidden_states=encoder_hidden_states,
+                cross_attention_kwargs=cross_attention_kwargs,
+                attention_mask=attention_mask,
+                encoder_attention_mask=encoder_attention_mask,
+                return_dict=False,
+            )[0]
```

Comment:
Are we not allowing gradient checkpointing or maybe I am misunderstanding the change?

Reply:
There was no gradient checkpointing here to begin with actually. Both if-branches had the same code so I moved them out. If we'd like to add gradient checkpointing support, let's do it in a separate PR?

Reply:
Ah okay. Makes perfect sense.